third-party-major.json

```diff
@@ -56,6 +56,12 @@
       "matchManagers": ["npm"],
       "matchPackageNames": ["dd-trace"],
 
+      "allowedVersions": "!/^3.42.0$/"
+    },
+    {
```

Comment:
Does having two separate rules and them together in the way we want here?

Reply:
Ah great point - I've merged them together to be safe